packages/mds-api-helpers/index.ts

```diff
@@ -55,11 +55,11 @@ export const asJsonApiLinks = (req: express.Request, skip: number, take: number,
   return undefined
 }
 
-export const parseQuery = <T = string>(query: { [k: string]: unknown }, parser?: (val: string) => T) => {
+export const parseQuery = <T = string>(req: express.Request, parser?: (val: string) => T) => {
```

Comment:
Don't need to do it in this PR -- but I can imagine the subset of this function that takes a generic map and lets you apply transformations for a destructure in a one-liner could be pretty cool

Reply:
`mds-utils` now has a `parseObjectProperties` method and `mds-api-helpers` wraps it with a `parseRequest` method that can be used to parse properties from `req.params` or `req.query`.